.gitignore

```diff
@@ -1,4 +1,3 @@
 *.gem
-Gemfile.lock
```

Comment:
I include the Gemfile.lock now because I've moved the build process to use Docker, so using the same .lock file makes sense since that the dev environments are the same